config/locales/devise.nl.yml

```diff
@@ -0,0 +1,63 @@
+# if me: DO NOT modify this file. if you need to add translations, add them to en.yml
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+nl:
+  devise:
+    confirmations:
+      confirmed: "Je e-mailadres is met succes bevestigd."
+      send_instructions: "Je zult binnen enkele minuten een e-mail ontvangen met instructies om je e-mailadres te bevestigen."
+      send_paranoid_instructions: "mail ontvangen met instructies om je account te activeren."
+    failure:
+      already_authenticated: "Je bent reeds ingelogd."
+      inactive: "Je account is nog niet geactiveerd."
+      invalid: "Invalide %{authentication_keys} of wachtwoord."
```

Comment:
Beetje laat om nog te nitpicken over "invalide accountgegevens". In goed Nederlands is dat natuurlijk "onjuiste ~" of "ongeldige ~".

Ahja. Volgende keer beter :-D

Mooi moment als alles naar pofiles omgezet wordt ;)

Reply:
Ik zet hem op de lijst met updates.